src/gemm_v2.cpp

```diff
@@ -324,41 +325,216 @@ inline void ProfilingRecordStop(const Handle& handle, HipEventPtr& start, HipEve
+    pref.setMaxWorkspaceBytes(0);
+
+    hipblaslt_ext::GemmEpilogue epilogue;
+    hipblaslt_ext::GemmTuning tuning;
+
+    hipblasOperation_t opTypeA = gemm_desc.transA ? HIPBLAS_OP_T : HIPBLAS_OP_N;
+    hipblasOperation_t opTypeB = gemm_desc.transB ? HIPBLAS_OP_T : HIPBLAS_OP_N;
+
+    hipblaslt_ext::GemmProblemType problemType;
+    problemType.op_a         = opTypeA;
+    problemType.op_b         = opTypeB;
+    problemType.type_a       = hip_type_AB;
+    problemType.type_b       = hip_type_AB;
+    problemType.type_c       = hip_type_C;
+    problemType.type_d       = hip_type_C;
+    problemType.type_compute = HIPBLAS_COMPUTE_32F;
+
+    hipblaslt_ext::Gemm gemm(handle.HipblasLtHandle().get(),
+                             opTypeA,
+                             opTypeB,
+                             hip_type_AB,
+                             hip_type_AB,
+                             hip_type_C,
+                             hip_type_C,
+                             HIPBLAS_COMPUTE_32F);
+
+    check_hipblas_status(gemm.setProblem(gemm_desc.m,
+                                         gemm_desc.n,
+                                         gemm_desc.k,
+                                         gemm_desc.batch_count,
+                                         gemm_desc.lda,
+                                         gemm_desc.ldb,
+                                         gemm_desc.ldc,
+                                         gemm_desc.ldc,
+                                         gemm_desc.strideA,
+                                         gemm_desc.strideB,
+                                         gemm_desc.strideC,
+                                         gemm_desc.strideC,
+                                         epilogue,
+                                         inputs,
+                                         problemType));
+
+    std::vector<hipblasLtMatmulHeuristicResult_t> heuristic;
+    check_hipblas_status(gemm.algoGetHeuristic(1, pref, heuristic));
+
+    check_hipblas_status(gemm.initialize(heuristic[0].algo, tuning, nullptr, handle.GetStream()));
+
+    {
+        HipEventProfiler profiler(handle);
+        check_hipblas_status(gemm.run(handle.GetStream()));
+    }
+}
+
+static void call_miopen_hipblasLt_gemm(const miopen::Handle& handle,
+                                       const miopen::GemmDescriptor& gemm_desc,
+                                       ConstData_t A,
+                                       std::size_t a_offset,
+                                       ConstData_t B,
+                                       std::size_t b_offset,
+                                       Data_t C,
+                                       std::size_t c_offset)
+{
+    MIOPEN_LOG_FUNCTION("hipBLASLt");
+
+    switch(gemm_desc.dataType)
+    {
+    case miopenInt8: {
+        MIOPEN_THROW(miopenStatusBadParm,
+                     "miopenInt8 type not supported for hipBlasLt GemmBackend");
```

Comment:
```suggestion
                     "the miopenInt8 type is not supported for hipBlasLt");
```
or simply
```C++
                     "miopenInt8 is not supported for hipBlasLt");
```
Ditto all other places.